src/calendar/view/EventPreviewView.js

```diff
@@ -18,15 +18,31 @@ export type Attrs = {
 export class EventPreviewView implements MComponent<Attrs> {
 
 	view({attrs: {event, limitDescriptionHeight, sanitizedDescription}}: Vnode<Attrs>): Children {
+
+		const location = event.location.trim()
```

Comment:
I hope it's not too slow to parse this URL each time. We could cache it with `memoize()`